semantics/executable-spec/src/Control/Provenance.hs

```diff
@@ -141,7 +141,7 @@ modifyMState delta = do
   mstore <- get
   case mstore of
     SNothing -> pure ()
-    (SJust st) -> put (SJust (delta st))
```

Comment:
Line 66 here is `import Control.Monad.State.Strict`, so I suspect this change is unnecessary.

Reply:
I confirmed with the threadnet tests that that annotation was indeed unnecessary.